src/ziggurat/middleware/default.clj

```diff
@@ -30,7 +30,9 @@
           (sentry/report-error sentry-reporter e (str "Couldn't parse the message with proto - " proto-class))
           (metrics/multi-ns-increment-count multi-namespaces "failed" additional-tags)
           nil)))
-    message))
+    (let [{left :left right :right} message]
```

Comment:
why not use the `:keys` destructuring syntax here

Reply:
yep will be doing that